resources/views/components/spinner.blade.php

```diff
@@ -0,0 +1 @@
+<x-butler::icons.loading {{ $attributes->class(['animate-spin']) }}/>
```

Comment:
default color that can be overwritten?

Reply:
That should be possible by adding a `class` or `style` attribute on the component?